content/desktop/extensions-sdk/guides/oauth2-flow.md

```diff
@@ -69,7 +69,7 @@ Next, you exchange the authorization code for an access token.
 
 The extension must send a `POST` request to the oauth authorization server with the following parameters:
 
-```
+```text
```

Comment:
Looks like it supports `http` highlighting; have you tried if that works? https://gohugo.io/content-management/syntax-highlighting/